frontend/src/metabase/visualizations/shared/components/RowChart/utils/data.ts

```diff
@@ -0,0 +1,180 @@
+import _ from "underscore";
+import { stack, stackOffsetDiverging, stackOffsetExpand } from "d3-shape";
+import type { Series as D3Series } from "d3-shape";
+import d3 from "d3";
```

Comment:
`d3` and `d3-xxx` are of different (major) versions atm. Should we also use `d3-xxx` package here, as visx does?